storagenode/storagenodedb/infodb.go

```diff
@@ -269,6 +272,31 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(piece_expiration) WHERE piece_expiration IS NOT NULL`,
 				},
 			},
+			{
+				Description: "Clear Tables from Alpha data",
+				Version:     11,
+				Action: migrate.SQL{
+					`DELETE FROM pieceinfo`,
+					`DELETE FROM used_serial`,
+					`DELETE FROM order_archive`,
+				},
+			},
+			{
+				Description: "Free Storagenodes from trash data",
+				Version:     12,
+				Action: migrate.Func(func(log *zap.Logger, mgdb migrate.DB, tx *sql.Tx) error {
+					deletecmd := exec.Command("rm", "-r",
+						fmt.Sprintf("%s/blob/%s", db.location, "ukfu6bhbboxilvt7jrwlqk7y2tapb5d2r2tsmj2sjxvw5qaaaaaa"), // us-central1
```

Comment:
Don't use `fmt.Sprintf`, only `filepath.Join` for path handling.

Reply:
good idea, let me do that.